src/xdocs/anttask.xml.vm

```diff
@@ -481,11 +463,6 @@
         of code convention violations.&quot; &gt;
   &lt;checkstyle config=&quot;checkstyle.xml&quot;
               failOnViolation=&quot;false&quot;&gt;
-    &lt;classpath&gt;
-      &lt;pathelement path=&quot;${classpath}&quot;/&gt;
-      &lt;pathelement location=&quot;lib/sevntu-checks-1.42.0.jar&quot;/&gt;
-    &lt;/classpath&gt;
```

Comment:
This is not 

>Run checkstyle with a custom JAR with implementation of custom Checks
          (example sevntu extension below)

Any more. Should we copy how we do this in examples repo? Or remove completely and give a link to that repo

Reply:
I trimmed back the description. I think it says more than an example would show.

Reply:
Some time ago you added this example as it was not clear even for you and me how to do this. Let's put here a link to examples project where example is shown

Reply:
Done.